src/index.html

```diff
@@ -309,13 +308,14 @@
                                 </div>
                             </div>
                         </div>
-                        <div>
-                            <input type="checkbox" checked data-toggle="toggle" data-on="Under 5s" data-off="All Ages"
-                                   data-bind="checked: allAges, visible: showAgeGroupCheck, click: () => debug('check box')">
-                        </div>
-<!--                         <div>
-                            <input data-bind="ageGroupToggleOn: active" type="checkbox" data-toggle="toggle"/>
-                        </div> -->
+                           <div class="btn-group mt-1" role="group">
+                                <button type="button" class="btn btn-primary" data-bind="click: () => changeAgeGroup('all'),
+                                 css: {active: ageGroup() == 'all'}">All Ages
+                                </button>
+                                <button type="button" class="btn btn-primary" data-bind="click: () => changeAgeGroup('under5'),
+                                 css: {active: ageGroup() == 'under5'}">Lifetime
+                                </button>
+                            </div>
                         <button type="button" class="btn btn-info btn-circle" data-toggle="modal" data-target="#methodModal">
```

Comment:
This help text now appears after the Ages button, but is actually the method help so needs to be moved up.